src/config/dynamic-theme-fixes.config

```diff
@@ -33101,6 +33101,16 @@ tp-yt-paper-menu-button {
 #paid-comment-background.ytd-comment-view-model {
     background-color: transparent !important;
 }
+svg[id^="yt-ringo2"] :not([id^="youtube-paths"]) path:nth-child(1) {
```

Comment:
Instead of forcing color in CSS move this selector to IGNORE INLINE STYLE then it will be using original color from youtube. 😉